grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/GrpcClientFactory.java

```diff
@@ -216,4 +238,29 @@ private static IllegalArgumentException newUnknownClientTypeException(Class<?> c
         }
         return ImmutableList.copyOf(descriptor.getMethods());
     }
+
+    private static Class<?> generatedStub(Annotation annotation, Method valueMethod) {
+        try {
+            final Object kClass = valueMethod.invoke(annotation, null);
+            return Class.forName(kClass.toString().substring("class ".length()));
```

Comment:
How about casting `Object` to `Class<?>`?
```suggestion
            final Class<?> kClass = (Class<?>) valueMethod.invoke(annotation, null);
            return Class.forName(kClass.getName());
```

Reply:
Why not just returning `kClass` then? No point of doing `Class.forName()`

Reply:
> Why not just returning `kClass` then? No point of doing `Class.forName()`

Ahh, you are right! 🤣

Reply:
Oops fixed. 😅